src/js/store/appState/reducers.ts

```diff
@@ -16,6 +17,7 @@ import {
 const initialState: AppState = {
   selectedLanguage: 'en',
   renderModal: '',
+  renderInfoModal: '',
```

Comment:
naming this `renderInfoModal` is a bit confusing. We are keeping layerID here for the info modal consumption. I think it should reflect that.